Makefile

```diff
@@ -1,6 +1,11 @@
 IMAGE_BUILD_CMD ?= "docker"
 REGISTRY_NAMESPACE ?= "ocs-dev"
 IMAGE_TAG ?= "latest"
+OPERATOR_SDK_VERSION="v0.10.0"
+OPERATOR_SDK_PLATFORM ?= "x86_64-linux-gnu"
+OPERATOR_SDK="operator-sdk-$(OPERATOR_SDK_VERSION)-$(OPERATOR_SDK_PLATFORM)"
+OUTPUT_DIR="_build/_output"
```

Comment:
I typoed this. it shouldn't have been `_build/_output`, it should have been `build/_output`

I'll send another PR in to fix this.